stylesheet.css

```diff
@@ -0,0 +1,22 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+.gridBoxes:hover {
+    box-shadow: 0 0 0.5em black;
```

Comment:
Please what does the "em" in "0.5em" mean?

Reply:
The em is a css measuring unit that equals the font size, but in this case it was an oversight i will change it cos it applies blur to the box.

Reply:
It has been adjusted sir thanks

Reply:
Thank you for the explanation.